Uploader.m

```diff
@@ -24,85 +24,25 @@ - (void)uploadFiles:(RNFSUploadParams*)params
   [req setHTTPMethod:method];
 
   // set headers
-  NSString *formBoundaryString = [self generateBoundaryString];
-  NSString *contentType = [NSString stringWithFormat:@"multipart/form-data; boundary=%@", formBoundaryString];
+  NSString *contentType = @"video/mp4";
```

Comment:
we will probably upload thumbnails too (we're sending a `Content-type` header in the @aacotroneo merge)